utils/cert.js

```diff
@@ -2,6 +2,51 @@ const crypto = require('crypto');
 const tools = require('./tools.js');
 const iota = require('./iotaSetup.js');
 
+// IOTA - Attach your transfer to the tangle
+const find = async (query) => {
+  const p = new Promise((res, rej) => {
+    iota.api.findTransactions(query, (e, s) => {
+      if (e) return rej(e);
+      res(s);
+    });
+  });
+  return p;
+};
+
+// IOTA - Attach your transfer to the tangle
```

Comment:
Is the comment perfectly describing the following lines?

Reply:
Code documentation updated.